content/03-reference/01-tools-and-interfaces/01-prisma-schema/05-models.mdx

```diff
@@ -8,9 +8,9 @@ metaDescription: 'Learn about defining your application models with Prisma. Mode
 
 This is an extension of the [Data model](data-model) page that discusses _models_ in the data model definition in detail.
 
-Models represent the entities of your application domain. They are defined using `model` blocks in the data model. In the [example data model](data-model#example), `User`, `Profile`, `Post` and `Category` are models. Here's the `User` model from the example on the again for reference:
+Models represent the entities of your application domain. They are defined using `model` blocks in the data model. In the [example data model](data-model#example), `User`, `Profile`, `Post` and `Category` are models.
 
-<a id="example" />
+#### Here's the `User` model from the example for reference {#example}
```

Comment:
```suggestion
Here's the `User` model from the example for reference {#example}:
```

Reply:
(This shouldn't be a title)

Reply:
Yup that's correct, **But** the problem here is we can only able to access **'id' value** of any headers as a link. I really thought about it, and then I created it as H4 header, since H5-H6 formats are not looking well, 

I think I got something now , I have to check it first. then I will let you know. give me little more time

Reply:
Now it works fine, Now we have both options, 
- Can use short custom links for any headers, Just provide it with custom-id, like `### header {#custom-id}` and use it like `[Mylink](#custom-id)`
- Can use `<a id="link-id"/>` and use it like `[Mylink](page-name#link-id)`